Source/DafnyLanguageServer.Test/Various/CounterExampleTest.cs

```diff
@@ -505,9 +505,9 @@ method listHasSingleElement(list:List<real>)
       Assert.AreEqual(1, counterExamples.Length);
       Assert.AreEqual(1, counterExamples[0].Variables.Count);
       Assert.IsTrue(counterExamples[0].Variables.ContainsKey("s:seq<bv5>"));
-      Assert.AreEqual("(Length := 2, [1] := 2bv5)", counterExamples[0].Variables["s:seq<bv5>"]);
+      Assert.AreEqual("(Length := 2, [1] := 2)", counterExamples[0].Variables["s:seq<bv5>"]);
```

Comment:
Are there improvements to counterexample generation in this PR that could be factored out in a PR separate from the test generation?